pkgs/development/python-modules/systembridge/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "systembridge";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "timmo001";
+    repo = "system-bridge-connector-py";
+    rev = "v${version}";
+    sha256 = "0ikdwn3a3w91ijmwgm4yjj1cz0bpnpkg2bf125x847sjh0hwc593";
+  };
+
+  buildInputs = [
```

Comment:
This seems wrong.

Reply:
Thanks, should be `propagatedBuildInputs`.